vaas/vaas/cluster/cluster.py

```diff
@@ -20,11 +20,9 @@
 
 
 @app.task(bind=True, soft_time_limit=settings.CELERY_TASK_SOFT_TIME_LIMIT_SECONDS)
-def load_vcl_task(self, emmit_time, cluster_ids):
-    emmit_time_aware = timezone.make_aware(datetime.strptime(emmit_time, "%Y-%m-%dT%H:%M:%S.%fZ"),
-                                           timezone=timezone.utc)
+def load_vcl_task(self, emmit_time: datetime, cluster_ids: List[int]) -> bool:
```

Comment:
this function is called as a celery task, so parameters are serialized and deserialized, are you sure that information about timezone is properly passed from caller to receiver?

Reply:
Yes.